src/components/ExecutorDetails.js

```diff
@@ -0,0 +1,311 @@
+            react: React,
+            reactRouterDom,
+            reactCore,
+            reactIcons,
+            pfReactTable
+        });
+
+        getRegistry().register({
+            ...mergeWithEntities(reducers.playbookActivityIntentory({
+                INVENTORY_ACTION_TYPES, renderStatus: (status) => (<div className="ins-c-remediations-status-bar">
+                    { renderStatus(normalizeStatus(status)) }
+                </div>)
+            })())
+        });
+
+        const { InventoryTable } = inventoryConnector();
+        setInventoryTable(() => InventoryTable);
+    };
+
+    const onRefresh = (options) => {
+        if (inventory && inventory.current) {
+            setPage(options.page);
+            setPageSize(options.per_page);
+            inventory.current.onRefreshData(options);
+        }
+    };
+
+    useEffect(() => {
+        loadInventory();
+        loadRemediation(id);
+        getPlaybookRun(id, run_id);
+
+    }, []);
+    useEffect(() => {
+        if (playbookRun && playbookRun.data) {
+            setExecutor(playbookRun.data.executors.find(executor => executor.executor_id === executor_id) || {});
+        }
+
+        getPlaybookRunSystems(id, run_id, executor_id);
+    }, [ playbookRun ]);
+
+    useEffect(() => {
+        setSystems(() => playbookRunSystems.map(({ system_id, system_name, status }) => ({
+            id: system_id,
+            display_name: system_name,
+            status,
+            children: <Skeleton size='lg' />
+        })));
+    }, [ playbookRunSystems ]);
+
+    useEffect(() => {
+        setSystems(() => ([
+            ...systems.filter(system => system.id !== playbookRunSystemDetails.system_id),
+            { ...systems.find(system => system.id === playbookRunSystemDetails.system_id),
+                isOpen: true,
+                children: <SyntaxHighlighter language="yaml" style={ dark }>
+                    { playbookRunSystemDetails && playbookRunSystemDetails.console || '' }
+                </SyntaxHighlighter>
+            }
+        ]));
+    }, [ playbookRunSystemDetails ]);
+
+    const systemsStatus = playbookRunSystems.reduce((acc, { status }) => ({ ...acc, [normalizeStatus(status)]: acc[normalizeStatus(status)] + 1 })
+        , { pending: 0,
+            running: 0,
+            success: 0,
+            failure: 0,
+            canceled: 0 });
+
+    const renderMain = (status) => ({
+        running: <Main>
+            <Stack gutter="md">
+                <Card>
+                    <CardHeader className='ins-m-card__header-bold'>
+                        <Button
+                            variant='link' onClick={ () => downloadPlaybook(remediation.id) }>
+                            Download Playbook
+                        </Button>
+                    </CardHeader>
+
+                    <CardBody>
+                        { InventoryTable && <InventoryTable
+                            ref={ inventory }
```

Comment:
The mocks have a "filter by name" search box. It's possible to filter hosts by name using the `ansible_host` query param in https://cloud.redhat.com/docs/api/remediations#operations-remediations-getPlaybookRunSystems